src/ts/implicateCartNumber.ts

```diff
@@ -0,0 +1,16 @@
+export function implicateCartNumber(){
+    const cartNumber = document.querySelector('.header__cart__number__content') as HTMLDivElement;
+    let cartProducts = [];

```

Comment:
redundant variable

Reply:
It isn't redundant. It has meaning anyway.
![request](https://user-images.githubusercontent.com/83092601/213488061-84bf0713-89d5-40be-8bff-1f1a36fc3404.JPG)

Reply:
```ts
export function implicateCartNumber(){
    const cartNumber = document.querySelector('.header__cart__number__content') as HTMLDivElement;
    const localStore = localStorage.getItem('cartProducts');
    if (localStore){
        const cartProducts: CartLocalStore[] = JSON.parse(localStore);
        const cartExample = cartProducts[0];
        const productNumbers: number = cartProducts.reduce(
            (sum: number, item: typeof cartExample) => sum + item.count, 0);

        cartNumber.textContent = String(productNumbers);
    } else {
        cartNumber.textContent = String(0);
    }
}
```

The main idea here is that you don't need to create an empty array and then assign its lengths (`0`).

Reply:
Ok, thanks) Fixed by https://github.com/PetrAlexkulakov/Online-Store/pull/18/commits/015f88ec1a479a0d47c4311db70c28b4f08c979f